Solutions/Forescout Host Property Monitor/Playbooks/ReadMe.md

```diff
@@ -0,0 +1,68 @@
+# ForeScout playbook templates
+
+<img src="./forescout-logo.png" alt="drawing" width="20%"/><br>
+
+## Table of Contents
+
+1. [Overview](#overview)
+1. [Playbook Templates](#playbooks)
+1. [Known issues and limitations](#limitations)
+
+<a name="overview">
+
+# Overview
+
+"Forescout Host Property Monitor" is a project that helps to check and report IP addresses involved in various categories of malicious attacks by analyzing host properties uploaded by Forescout Continuum platform. This project analyzes number of times certain events are generated by the host to determine the classificaction and severity of activity by the host. It forwards that severity information back to Forescout Continuum platform through description field fo Sentinel Incident so that Coninuum platform can act upon the host.
+
+<a name="playbooks">
+
+# Playbook Templates
+This package includes:  
```

Comment:
Please add deployment buttons of the playbook. Take reference from the example stated earlier.